subprojects/core/src/main/java/org/gradle/StartParameter.java

```diff
@@ -90,6 +81,7 @@
     private boolean continuous;
     private List<File> includedBuilds = new ArrayList<File>();
     private boolean buildScan;
+    private String bootstrapInitScriptUrl;
```

Comment:
We could also have two lists, one for provided init scripts(ones located in init.d folders) and user init scripts. But I like your idea better. I can take that path and fix this up using that until others chime in.